src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs

```diff
@@ -251,6 +253,8 @@ public T GetElement(int index)
         /// <returns>A <see cref="Vector128{T}" /> with the value of the element at <paramref name="index" /> set to <paramref name="value" /> and the remaining elements set to the same value as that in the current instance.</returns>
         /// <exception cref="NotSupportedException">The type of the current instance (<typeparamref name="T" />) is not supported.</exception>
         /// <exception cref="ArgumentOutOfRangeException"><paramref name="index" /> was less than zero or greater than the number of elements.</exception>
+        [Intrinsic]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I'm not sure that this is worth aggressively inlining the software fallback case.

Reply:
This `AggressiveInlining` is for "non-constant" fallback. Call-sites in heavy vector contexts is harmful to caller's register allocation due to the caller-saved SIMD registers (especially on Unix).

Reply:
Users that have non-constant fallbacks will already incur a perf hit, no matter which way it goes. The additional overhead of the bounds check, the additional (non-inlined call) for the throw, the stack store, etc probably make it non-worthwhile to aggressively inline this as well.

Reply:
I think we should still consider this and, if nothing else, show the assembly that is being inlined for the software fallback case.

Reply:
Yes, the codegen of inlined non-const fallback looks not good due to the exception code.

```csharp
            vf = vf.WithElement(index, -1.0f);
            if (vf.GetElement(index) != -1.0f)
            {
                return false;
            }
```
generates
```asm
IN0086: 000287 mov      rdi, 0x11528FE80
IN0087: 000291 mov      esi, 1
IN0088: 000296 call     CORINFO_HELP_GETSHARED_NONGCSTATIC_BASE
IN0089: 00029B mov      rax, 0x11528FEB4
IN008a: 0002A5 mov      eax, dword ptr [rax]
IN008b: 0002A7 cmp      eax, 4
IN008c: 0002AA jae      SHORT G_M61425_IG24

G_M61425_IG18:        ; offs=0002ACH, size=003EH, gcrefRegs=00000000 {}, byrefRegs=00000000 {}, byref, isz

IN008d: 0002AC movaps   xmm0, xmmword ptr [V04 rbp-30H]
IN008e: 0002B0 movaps   xmmword ptr [V38 rbp-A0H], xmm0
IN008f: 0002B7 lea      rdi, bword ptr [V38 rbp-A0H]
IN0090: 0002BE movsxd   rax, eax
IN0091: 0002C1 movss    xmm0, dword ptr [reloc @RWD24]
IN0092: 0002C9 movss    dword ptr [rdi+4*rax], xmm0
IN0093: 0002CE movaps   xmm0, xmmword ptr [V38 rbp-A0H]
IN0094: 0002D5 movaps   xmmword ptr [V04 rbp-30H], xmm0
IN0095: 0002D9 mov      rax, 0x11528FEB4
IN0096: 0002E3 mov      eax, dword ptr [rax]
IN0097: 0002E5 cmp      eax, 4
IN0098: 0002E8 jae      SHORT G_M61425_IG25

G_M61425_IG19:        ; offs=0002EAH, size=0017H, gcrefRegs=00000000 {}, byrefRegs=00000000 {}, byref, isz

IN0099: 0002EA lea      rdi, bword ptr [V04 rbp-30H]
IN009a: 0002EE movsxd   rax, eax
IN009b: 0002F1 movss    xmm0, dword ptr [rdi+4*rax]
IN009c: 0002F6 ucomiss  xmm0, dword ptr [reloc @RWD28]
IN009d: 0002FD jp       SHORT G_M61425_IG20
IN009e: 0002FF je       SHORT G_M61425_IG22

G_M61425_IG20:        ; offs=000301H, size=0002H, gcrefRegs=00000000 {}, byrefRegs=00000000 {}, byref

IN009f: 000301 xor      eax, eax

G_M61425_IG21:        ; offs=000303H, size=0006H, epilog, nogc, emitadd

IN00ae: 000303 lea      rsp, [rbp]
IN00af: 000307 pop      rbp
IN00b0: 000308 ret      

G_M61425_IG22:        ; offs=000309H, size=0005H, gcVars= {}, gcrefRegs=00000000 {}, byrefRegs=00000000 {}, gcvars, byref

IN00a0: 000309 mov      eax, 1

G_M61425_IG23:        ; offs=00030EH, size=0006H, epilog, nogc, emitadd

IN00b1: 00030E lea      rsp, [rbp]
IN00b2: 000312 pop      rbp
IN00b3: 000313 ret      

G_M61425_IG24:        ; offs=000314H, size=000BH, gcVars= {}, gcrefRegs=00000000 {}, byrefRegs=00000000 {}, gcvars, byref

IN00a1: 000314 mov      edi, 15
IN00a2: 000319 call     ThrowHelper:ThrowArgumentOutOfRangeException(int)
IN00a3: 00031E int3     

G_M61425_IG25:        ; offs=00031FH, size=000BH, gcrefRegs=00000000 {}, byrefRegs=00000000 {}, byref

IN00a4: 00031F mov      edi, 15
IN00a5: 000324 call     ThrowHelper:ThrowArgumentOutOfRangeException(int)
IN00a6: 000329 int3     
```

Reply:
Removed `AggressiveInlining`.